### DISCUSSION TITLE
can't get PHP_PHPCS working to auto fix minor things with phpcbf

### DISCUSSION THREAD

**Discussion by rzewus** [oxsecurity/megalinter#1930] [Q&A] [answered]
Hello there ! 
I am trying to find some "switch" or env variable to enable phpcs corrections, but can't seem to find it.
Shouldn't it be available ?! :)
or should I run separate GitHub Action to correct that?

Marvellous work btw :) possibility to have all in one place really is convenient :D

**Comment by llaville** [comment, marked as answer]
Hello @rzewus 

The reason is very simple : MegaLinter v5 or v6 (don't know which version you are running) embed only PHPCS binary **and not** PHPCBF binary.

It may be quite simple to add it into MegaLinter, because `Phive` (installation method for PHPCS; see https://github.com/oxsecurity/megalinter/blob/v6.11.1/megalinter/descriptors/php.megalinter-descriptor.yml#L61-L64) also provide a way to install PHPCBF (known in repositories list; see https://github.com/phar-io/phar.io/blob/master/data/repositories.xml#L66-L68

Differences between both of binary are explained (for those who don't know yet) on official PHPCS Wiki at https://github.com/squizlabs/PHP_CodeSniffer/wiki/Fixing-Errors-Automatically.

In future report/question, please provide versions you're running and platform. It may help those who can help you !

**Comment by rzewus** [comment]
I am using latest version :)
THANK YOU VERY MUCH for pointing me in right direction! :)

**Comment by llaville** [comment]
As PHPCBF is not part of standard MegaLinter distribution, I suggest to add it via Plugins feature (read https://oxsecurity.github.io/megalinter/latest/plugins/)
